system/lib/wasmfs/virtual.h

```diff
@@ -15,7 +15,7 @@
 
 #pragma once
 
-#include "file.h"
+#include <emscripten/wasmfs_internal/file.h>
```

Comment:
I would hope we could keep `wasmfs_internal` headers in the `system/lib/wasmfs` directory and then have them installed via the `install_system_headers` help in `tools/system_libs.py`.  No need to relocate them in git to `system/include` I would hope.